oak_remote_attestation/src/dice.rs

```diff
@@ -237,12 +240,3 @@ impl LayerEvidence {
         get_claims_set_from_certificate_bytes(&self.eca_certificate).map_err(anyhow::Error::msg)
     }
 }
-
-/// Extracts the bytes used to encode a CBOR object from a slice that might include unused bytes by
-/// deserializing and re-serializing the encoded CBOR object.
-fn cbor_encoded_bytes_to_vec(bytes: &[u8]) -> anyhow::Result<Vec<u8>> {
-    let mut result = Vec::new();
-    let value: Value = from_reader(bytes).map_err(anyhow::Error::msg)?;
-    into_writer(&value, &mut result).map_err(anyhow::Error::msg)?;
-    Ok(result)
-}
```

Comment:
down the road, we it will make sense simplify this file further by encoding & shared decoding logic for the dice data to the oak_dice crate.

For now this just introduces the minimal amount of sharing needed to prevent code duplication for this specific PR.